client-libs/python/pyproject.toml

```diff
@@ -13,7 +13,7 @@ python = "^3.8"
 httpx = "^0.24.1"
 attrs = "^23.1.0"
 python-dateutil = "^2.8.2"
-openai = "~1.7.0"
+openai = ">= 1.7, <= 1.12"
```

Comment:
You probably mean: 
```suggestion
openai = ">= 1.7, < 1.13"
```
Otherwise, you'd be stuck with `1.12.0` (and not get patch versions like `1.12.1`).

Reply:
There are definitely tradeoffs here. In the past OpenAI has released patch versions that actually broke our ingestion workflow (breaking changes aren't guaranteed to be minor or major version updates). But since OpenAI's SDK has been reasonably stable for a couple months, I'm willing to take a chance and include `1.12` patch versions!

Let me update the dependency requirements.